projects/shared/src/lib/component/documents/document-description/document-description.component.html

```
@@ -321,12 +317,12 @@
   }
 
   <!-- SERIES STATEMENT -->
-  @if (record().metadata.seriesStatement) {
+  @if (seriesStatement().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)